### PR TITLE
docs(ha): Add section explaining how to delete orphan services.

### PR DESCRIPTION
**Pull request by joonlim**

### CODE REVIEW

**Comment by joonlim** [comment]
@dorbin @lwander @jtk54

File: reference/halyard/high-availability.md [hunk 1]
```diff
@@ -135,6 +135,14 @@ This service's name when [configuring its sizing](/reference/halyard/component-s
 
 To add a [custom profile](/reference/halyard/custom/#custom-profiles) or [custom service settings](/reference/halyard/custom/#custom-service-settings) for this service, use the name `echo-worker`.
 
+## Deleting Orphan Services
```
Comment (lwander) [inline reference/halyard/high-availability.md]: I would call these "orphaned" services

Comment (dorbin) [reply reference/halyard/high-availability.md]: including in the option `--delete-orphaned-services`?

File: reference/halyard/high-availability.md [hunk 2]
```diff
@@ -135,6 +135,14 @@ This service's name when [configuring its sizing](/reference/halyard/component-s
...
+
+When enabling or disabling HA for a service on a running Spinnaker, Halyard will not clean up the old service(s) by default. This means that if a non-HA Clouddriver is running and Spinnaker is then deployed with HA Clouddriver enabled, the non-HA Clouddriver will still be running, even though it is no longer used. To clean up these orphan services, add a `--delete-orphan-services` flag to `hal deploy apply`:
```
Comment (jtk54) [inline reference/halyard/high-availability.md]: Update here if you change the flag name.

**Review by jtk54 — APPROVED** [review]

File: reference/halyard/high-availability.md (hunk 2 above)
Comment (dorbin) [inline reference/halyard/high-availability.md]: But not just Clouddriver, right? Can be any service? If so, I would add a "for example" to the above:

"This means that if a non-HA Clouddriver is running (for example)  and Spinnaker is then deployed..."

Comment (dorbin) [inline reference/halyard/high-availability.md]: The first sentence seems to  contradict the rest of the description. Mostly that's probably my lack of understanding, but if Halyard is going to clean up old services, I would not expect the non-HA Clouddriver to be still running. Could you clarify?

**Review by lwander — APPROVED** [review]

File: reference/halyard/high-availability.md (hunk 2 above)
Comment (dorbin) [reply reference/halyard/high-availability.md]: nm, I read "Halyard will clean up" rather than "Halyard will not clean up."

Carry on.

**Review by dorbin — APPROVED** [review]

**Pull request merged**